vendor/github.com/openshift/installer/pkg/asset/bootstraplogging/bootstraplogging.go

```diff
@@ -12,6 +12,7 @@ const (
 type Config struct {
 	Certificate       string
 	Key               string
+	Account           string
```

Comment:
No no :) Lets not do this :)

Please fix a change in `https://github.com/mjudeikis/installer/blob/release-4.7-azure`.
You will need to get this branch check out, find commit which adds bootstrap logging and "edit" individual commit to add this change in there and make a PR into our Installer fork. 

This not gonna fly :)

https://github.com/mjudeikis/installer/blob/release-4.7-azure/pkg/asset/bootstraplogging/bootstraplogging.go#L14

Reply:
@mjudeikis I reverted this particular change - parameterizing the log account name in the bootstrapper logging is beyond the scope of this change.

Reply:
But how this will work in Bootstrap logs in FF? This will break it?